src/main/java/de/thetaphi/forbiddenapis/maven/AbstractCheckMojo.java

```diff
@@ -363,11 +367,16 @@ public void info(String msg) {
         if (signaturesFiles != null) {
           sigFiles.addAll(Arrays.asList(signaturesFiles));
         }
+
         if (signaturesArtifacts != null) {
           for (final SignaturesArtifact artifact : signaturesArtifacts) {
             final File f = resolveSignaturesArtifact(artifact);
             if (artifact.path != null) {
-              sigUrls.add(createJarUrl(f, artifact.path));
+              if (f.isDirectory()) {
+                sigUrls.add(createFileUrl(f, artifact.path));
```

Comment:
I think, you could simply add the resolved file to sigFiles. This makes output for plain files nicer (it does not log an url, but a filename). It would simply handle the artifact as a plain file.

I am still confused, how this can work. It returns a directory instead of a file after artifact resolve????